contracts/Inbox.sol

```diff
@@ -102,7 +102,7 @@ contract Inbox is IInbox, Eco7683DestinationSettler, Semver {
         bytes32[] memory hashes = new bytes32[](1);
         address[] memory claimants = new address[](1);
         hashes[0] = _expectedHash;
-        claimants[0] = _claimant;
+        claimants[0] = TypeCasts.bytes32ToAddress(_claimant);
```

Comment:
We need to change this to accept Solana addresses. We shouldn't cast bytes32 to address here. We may only want to do this when withdrawing the rewards or refunding intents. SevenLabs requests this change.

Reply:
the claimants  array is unused so it shouldn't create an issue for seven labs. Maybe the issue is in initiateProving where we're typecasting from bytes32 to address again and pass it to IProver.prove()?

Reply:
fixed now with https://github.com/eco/eco-routes/pull/268